statsmodels/survey/tests/test_summary_stats.py

```diff
@@ -0,0 +1,18 @@
+import summary_stats
+import unittest
+import pandas as pd
+import numpy as np
+
+# df = pd.read_csv("examples/survey_df.csv")
```

Comment:
I wouldn't worry about the example now.  It will be more useful to have an example when the API is more solid and several methods are implemented.  When we do have an example, I think it should use a real survey dataset.  There are a bunch out there, but MEPS is one that I am familiar with.